src/lib/network-recorder.js

```diff
@@ -54,42 +52,37 @@ class NetworkRecorder {
   }
 
   onRequestServedFromCache(data) {
-    this._rawEvents.push({method: 'Network.requestServedFromCache', params: data});
     this.networkManager._dispatcher.requestServedFromCache(data.requestId);
   }
 
   onResponseReceived(data) {
-    this._rawEvents.push({method: 'Network.responseReceived', params: data});
     // NOTE: data.timestamp -> time, data.type -> resourceType
     this.networkManager._dispatcher.responseReceived(data.requestId,
         data.frameId, data.loaderId, data.timestamp, data.type, data.response);
   }
 
   onDataReceived(data) {
-    this._rawEvents.push({method: 'Network.dataReceived', params: data});
     // NOTE: data.timestamp -> time
     this.networkManager._dispatcher.dataReceived(data.requestId, data.timestamp,
         data.dataLength, data.encodedDataLength);
   }
 
   onLoadingFinished(data) {
-    this._rawEvents.push({method: 'Network.loadingFinished', params: data});
     // NOTE: data.timestamp -> finishTime
     this.networkManager._dispatcher.loadingFinished(data.requestId,
         data.timestamp, data.encodedDataLength);
   }
 
   onLoadingFailed(data) {
-    this._rawEvents.push({method: 'Network.loadingFailed', params: data});
     // NOTE: data.timestamp -> time, data.type -> resourceType,
     // data.errorText -> localizedDescription
     this.networkManager._dispatcher.loadingFailed(data.requestId,
         data.timestamp, data.type, data.errorText, data.canceled,
         data.blockedReason);
   }
 
-  onResourceChangedPriority(data) {
-    this._rawEvents.push({method: 'Network.resourceChangedPriority', params: data});
+  onResourceChangedPriority() {
+    // TODO: Add call to dispatcher after devtools dependency is updated
```

Comment:
not sure how to interpret what this TODO is asking for, but make sure to file an issue for this :)